sniffles/snifflesconfig.py

```diff
@@ -386,12 +395,13 @@ def getWriteRegEx(self):
 
     def setWriteRegExe(self, value):
         self.write_reg_ex = value
-
+    
```

Comment:
Do not add space characters to an empty line. They only increase diff without adding any value to the code.